src/components/Dashboard/TodoList.vue

```diff
@@ -197,7 +197,26 @@
                   <span>{{ item.subtitle }}</span>
                 </div>
 
-                <!-- draft other (continuation subtitle is handled separately) -->
+                <!-- draft continuation filing -->
+                <div
+                  v-else-if="EnumUtilities.isStatusDraft(item) && EnumUtilities.isTypeContinuationIn(item)"
+                  class="todo-list-detail pt-1"
+                >
+                  <span
+                    v-if="getNameRequest"
+                    class="list-item__subtitle"
+                  >
+                    Name Request APPROVED - {{ expiresText(getNameRequest) }}
+                  </span>
+                  <span
+                    v-else
+                    class="list-item__subtitle"
+                  >
+                    DRAFT
+                  </span>
+                </div>
+
+                <!-- draft other -->
                 <div
                   v-else-if="EnumUtilities.isStatusDraft(item) && !EnumUtilities.isTypeContinuationIn(item)"
```

Comment:
You can remove the continuation in condition (already handled by new line 202).

Reply:
Thanks, removed